pkg/vm/vm.go

```diff
@@ -1458,9 +1458,16 @@ func (v *VM) Jump(ctx *Context, offset int) {
 	ctx.nextip = offset
 }
 
-// Call calls method by offset. It is similar to Jump but also
-// pushes new context to the invocation state
-func (v *VM) Call(ctx *Context, offset int) {
+// CallAndIncrementInvocations calls method by offset. It is similar to Jump but also
+// pushes new context to the invocation state and increments invocation counter
+// for the corresponding context script hash.
+func (v *VM) CallAndIncrementInvocations(ctx *Context, offset int) {
```

Comment:
Let's call it `Invoke` then, easier and valid. Or even `Call` (with internal `call`) as non-incrementing version is only being used by `vm` internally.